client/components/command-palette/test/index.tsx

```diff
@@ -62,122 +68,21 @@ describe( 'CommandPalette', () => {
-			overlayElement && fireEvent.click( overlayElement );
-		} );
-
-		waitFor( () => {
-			expect( screen.queryByPlaceholderText( 'Search for commands' ) ).toBeNull();
-		} );
-	} );
-
-	it( 'should close the palette when you select a specific command', () => {
-		renderCommandPalette();
-
-		waitFor( () => {
-			expect( screen.getByPlaceholderText( 'Search for commands' ) ).toBeInTheDocument();
-			const getHelpCommand = screen.getByText( 'Get help' );
-			fireEvent.click( getHelpCommand );
-		} );
-
-		waitFor( () => {
-			expect( screen.queryByPlaceholderText( 'Search for commands' ) ).toBeNull();
-		} );
-	} );
-
-	it( 'should return only "Get help" command as it matches label and "Send feedback" as it matches searchLabel; other commands are hidden', () => {
-		renderCommandPalette();
-
-		waitFor( () => {
-			expect( screen.getByPlaceholderText( 'Search for commands' ) ).toBeInTheDocument();
-			fireEvent.change( screen.getByPlaceholderText( 'Search for commands' ), {
-				target: { value: 'help' },
-			} );
-		} );
 
-		waitFor( () => {
-			expect( screen.getByText( 'Get help' ) ).toBeInTheDocument();
-			expect( screen.getByText( 'Send feedback' ) ).toBeInTheDocument();
-			expect( screen.queryByText( 'Clear cache' ) ).toBeNull();
-			expect( screen.queryByText( 'Enable edge cache' ) ).toBeNull();
-		} );
-	} );
-
-	it( 'should return "No results found" when there is no match for search', () => {
-		renderCommandPalette();
-
-		waitFor( () => {
-			expect( screen.getByPlaceholderText( 'Search for commands' ) ).toBeInTheDocument();
-			fireEvent.change( screen.getByPlaceholderText( 'Search for commands' ), {
-				target: { value: 'blue' },
-			} );
-		} );
-
-		waitFor( () => {
-			expect( screen.getByText( 'No results found.' ) ).toBeInTheDocument();
-			expect( screen.getByText( 'Send feedback' ) ).toBeNull();
-			expect( screen.queryByText( 'Clear cache' ) ).toBeNull();
-			expect( screen.queryByText( 'Enable edge cache' ) ).toBeNull();
+		act( () => {
+			fireEvent.keyDown( document, { key: 'k', metaKey: true } );
 		} );
-	} );
+	};
 
-	it( 'should navigate to a nested command and back to root commands', () => {
+	it( 'should confirm that the command palette opens with the commands from the commands array', () => {
 		renderCommandPalette();
 
-		waitFor( () => {
-			expect( screen.getByPlaceholderText( 'Search for commands' ) ).toBeInTheDocument();
-			act( () => {
-				fireEvent.click( screen.getByText( 'Clear cache' ) );
-			} );
-		} );
-
-		waitFor( () => {
-			expect( screen.getByLabelText( 'Go back to the previous screen' ) ).toBeInTheDocument();
-			act( () => {
-				fireEvent.click( screen.getByLabelText( 'Go back to the previous screen' ) );
-			} );
-		} );
-
-		waitFor( () => {
-			expect( screen.queryByPlaceholderText( 'Search for commands' ) ).toBeInTheDocument();
-		} );
+		expect( screen.getByPlaceholderText( 'Search for commands' ) ).toBeInTheDocument();
+		expect( screen.getByText( 'Get help' ) ).toBeInTheDocument();
```

Comment:
Nice! Now this test is working correctly for me.

Reply:
Thanks! I fixed another case with search but I am running into some issues with testing closing the modal. I will continue investigating what is happening there